### PR TITLE
[POAE7-1437] llvm patch file

### PR DESCRIPTION
**Pull request by sijialouintel**
The modification of the original patch file is treated as garbage when patching

### CODE REVIEW

File: scripts/llvm-9-glibc-2.31-708430.patch [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-#if !defined(__aarch64__) || !SANITIZER_LINUX || __GLIBC_PREREQ(2, 21)
-/* On aarch64 glibc 2.20 and earlier provided incorrect mode field.  */
-- CHECK_SIZE_AND_OFFSET(ipc_perm, mode);
-+  // CHECK_SIZE_AND_OFFSET(ipc_perm, mode);
-#endif
-
-    CHECK_TYPE_SIZE(shmid_ds);
+diff --git a/sanitizer_platform_limits_posix.cc.orig b/sanitizer_platform_limits_posix.cc
+index b7fa6e8..2c42ece 100644
+--- compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc.orig
```
Comment (winningsix) [inline scripts/llvm-9-glibc-2.31-708430.patch:3]: hmm, QQ: why do we need this update to LLVM patch?

Comment (sijialouintel) [reply scripts/llvm-9-glibc-2.31-708430.patch:3]: Omnisci use this file to patch llvm 9 for glibc 2.31+ support. The original patch file can't be parsed successfully and are considered as garbage. I use 'git diff' to regenerate this patch file, which can be used by  
`patch -p0 < $SCRIPTS_DIR/llvm-9-glibc-2.31-708430.patch`  
(line 193 in https://github.com/Intel-bigdata/omniscidb/blob/cider/scripts/common-functions.sh)

**Comment by winningsix** [comment]
can we merge this into upstream Omnisci as well? @sijialouintel

**Comment by sijialouintel** [comment]
> can we merge this into upstream Omnisci as well? @sijialouintel

Yes, I think it should be merged into master. @winningsix

**Pull request merged**